build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.1.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.93.0'
+    gutenbergMobileVersion = '5690-0840f356a63edf3ded8a1df897ffedf997ff9ada'
```

Comment:
Note that this reference doesn't point to the last commit (https://github.com/wordpress-mobile/gutenberg-mobile/pull/5690/commits/5855a75e05102f6b264a183b846de4be899406cb). However, since the last changes only modified E2E tests, seems safe to use the current installable build to run smoke tests.

Reply:
Exactly, thanks for noticing it. Since the builds were already done for that commit I avoided pushing a change and having to wait again to smoke test 😄